packages/celotool/src/lib/k8s-oracle/rbac.ts

```diff
@@ -0,0 +1,69 @@
+import { installGenericHelmChart, removeGenericHelmChart, upgradeGenericHelmChart } from 'src/lib/helm_deploy'
+import { execCmdWithExitOnFailure } from '../cmd-utils'
+import { BaseOracleDeployer } from './base'
+
+// Oracle RBAC------
+// We need the oracle pods to be able to change their label to accommodate
+// limitations in aad-pod-identity & statefulsets (see https://github.com/Azure/aad-pod-identity/issues/237#issuecomment-611672987)
+// To do this, we use an auth token that we get using the resources in the `oracle-rbac` chart
```

Comment:
If I understood correctly, the rbac chart is decoupled from oracle chart to create 1 service account each oracle replica, thus being able to pass the SA identities/tokens from the rbac chart as values to the oracle chart. 
And in the oracle chart all the pods from the oracle statefulset have all the tokens as env variables, but each pod uses a different token based on their $RID. Is that right?

Reply:
Yup! That is correct